src/map/lua/lua_item.cpp

```diff
@@ -203,6 +203,21 @@ inline int32 CLuaItem::getAugment(lua_State* L)
     return 2;
 }
 
+int32 CLuaItem::getAppraisalID(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PLuaItem == nullptr);
+
+    auto PItem = dynamic_cast<CItem*>(m_PLuaItem);
+    if (PItem)
+    {
+        lua_pushinteger(L, PItem->m_extra[0x16]);
+    }
+    else
+        lua_pushinteger(L, 0);
```

Comment:
Curly braces since it follows an if with curly braces.
I would prefer lua_pushnil(L) in this case.